src/dotnet/Mgmt.CI.BuildTools/ci.common.props

```diff
@@ -0,0 +1,38 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
+		<VersionPrefix>0.9.0</VersionPrefix>
+		<AsmVersion>0.9.0</AsmVersion>
+		<AsmFileVersion>0.9.0</AsmFileVersion>
```

Comment:
Looks like CI it is creating a package for each of our projects as well as your root mgmtci.proj. To prevent that I think we should set `<IsPackable>false</IsPackable>` here so that all these don't generate packages.